<html>
<head>
  <title>VEP summary</title>
  <script type="text/javascript" src="http://www.google.com/jsapi"></script>
  <script type="text/javascript">
    google.load('visualization', '1', {packages: ['corechart','table']});
  </script>
  <script type="text/javascript">
    
    function init() {
      // charts
      var var_class_pie = drawPie('var_class_pie', 'Variant classes', google.visualization.arrayToDataTable([['Variant class','Count'],['insertion',6],['SNV',652408]]), null);
var var_class_table = drawTable('var_class_table', 'Variant classes', google.visualization.arrayToDataTable([['Variant class','Count'],['insertion',6],['SNV',652408]]));

          google.visualization.events.addListener(var_class_pie, 'select', function() {
            var_class_table.setSelection(var_class_pie.getSelection());
          });
          google.visualization.events.addListener(var_class_table, 'select', function() {
            var_class_pie.setSelection(var_class_table.getSelection());
          });
        var var_cons_pie = drawPie('var_cons_pie', 'Consequences (most severe)', google.visualization.arrayToDataTable([['Consequence type','Count'],['splice_donor_variant',135],['splice_acceptor_variant',150],['stop_gained',395],['stop_lost',213],['start_lost',31],['missense_variant',6885],['splice_region_variant',1084],['stop_retained_variant',56],['synonymous_variant',2669],['mature_miRNA_variant',21],['5_prime_UTR_variant',2271],['3_prime_UTR_variant',8154],['non_coding_transcript_exon_variant',10505],['intron_variant',340143],['upstream_gene_variant',30773],['downstream_gene_variant',25195],['intergenic_variant',223734]]), {slices: [{color: "#ff7f50"}, {color: "#ff7f50"}, {color: "#ff0000"}, {color: "#ff0000"}, {}, {color: "#ffd700"}, {color: "#ff7f50"}, {color: "#76ee00"}, {color: "#76ee00"}, {}, {}, {}, {}, {color: "#02599c"}, {color: "#a2b5cd"}, {color: "#a2b5cd"}, {color: "gray"}]});
var var_cons_table = drawTable('var_cons_table', 'Consequences (most severe)', google.visualization.arrayToDataTable([['Consequence type','Count'],['splice_donor_variant',135],['splice_acceptor_variant',150],['stop_gained',395],['stop_lost',213],['start_lost',31],['missense_variant',6885],['splice_region_variant',1084],['stop_retained_variant',56],['synonymous_variant',2669],['mature_miRNA_variant',21],['5_prime_UTR_variant',2271],['3_prime_UTR_variant',8154],['non_coding_transcript_exon_variant',10505],['intron_variant',340143],['upstream_gene_variant',30773],['downstream_gene_variant',25195],['intergenic_variant',223734]]));

          google.visualization.events.addListener(var_cons_pie, 'select', function() {
            var_cons_table.setSelection(var_cons_pie.getSelection());
          });
          google.visualization.events.addListener(var_cons_table, 'select', function() {
            var_cons_pie.setSelection(var_cons_table.getSelection());
          });
        var consequences_pie = drawPie('consequences_pie', 'Consequences (all)', google.visualization.arrayToDataTable([['Consequence type','Count'],['stop_gained',379],['stop_lost',208],['start_lost',18],['missense_variant',6842],['splice_region_variant',337],['stop_retained_variant',62],['synonymous_variant',2862],['NMD_transcript_variant',89]]), {slices: [{color: "#ff0000"}, {color: "#ff0000"}, {}, {color: "#ffd700"}, {color: "#ff7f50"}, {color: "#76ee00"}, {color: "#76ee00"}, {}]});
var consequences_table = drawTable('consequences_table', 'Consequences (all)', google.visualization.arrayToDataTable([['Consequence type','Count'],['stop_gained',379],['stop_lost',208],['start_lost',18],['missense_variant',6842],['splice_region_variant',337],['stop_retained_variant',62],['synonymous_variant',2862],['NMD_transcript_variant',89]]));

          google.visualization.events.addListener(consequences_pie, 'select', function() {
            consequences_table.setSelection(consequences_pie.getSelection());
          });
          google.visualization.events.addListener(consequences_table, 'select', function() {
            consequences_pie.setSelection(consequences_table.getSelection());
          });
        var coding_pie = drawPie('coding_pie', 'Coding consequences', google.visualization.arrayToDataTable([['Consequence type','Count'],['stop_gained',379],['stop_lost',208],['start_lost',18],['missense_variant',6842],['stop_retained_variant',62],['synonymous_variant',2862]]), {slices: [{color: "#ff0000"}, {color: "#ff0000"}, {}, {color: "#ffd700"}, {color: "#76ee00"}, {color: "#76ee00"}]});
var coding_table = drawTable('coding_table', 'Coding consequences', google.visualization.arrayToDataTable([['Consequence type','Count'],['stop_gained',379],['stop_lost',208],['start_lost',18],['missense_variant',6842],['stop_retained_variant',62],['synonymous_variant',2862]]));

          google.visualization.events.addListener(coding_pie, 'select', function() {
            coding_table.setSelection(coding_pie.getSelection());
          });
          google.visualization.events.addListener(coding_table, 'select', function() {
            coding_pie.setSelection(coding_table.getSelection());
          });
        var sift_pie = drawPie('sift_pie', 'SIFT summary', google.visualization.arrayToDataTable([['Prediction','Count'],['tolerated_low_confidence',272],['deleterious_low_confidence',329],['tolerated',1785],['deleterious',2433]]), {slices: [{}, {}, {color: "green"}, {color: "red"}]});
var sift_table = drawTable('sift_table', 'SIFT summary', google.visualization.arrayToDataTable([['Prediction','Count'],['tolerated_low_confidence',272],['deleterious_low_confidence',329],['tolerated',1785],['deleterious',2433]]));

          google.visualization.events.addListener(sift_pie, 'select', function() {
            sift_table.setSelection(sift_pie.getSelection());
          });
          google.visualization.events.addListener(sift_table, 'select', function() {
            sift_pie.setSelection(sift_table.getSelection());
          });
        var polyphen_pie = drawPie('polyphen_pie', 'PolyPhen summary', google.visualization.arrayToDataTable([['Prediction','Count'],['unknown',113],['possibly_damaging',912],['probably_damaging',1668],['benign',2260]]), {slices: [{color: "blue"}, {}, {}, {color: "green"}]});
var polyphen_table = drawTable('polyphen_table', 'PolyPhen summary', google.visualization.arrayToDataTable([['Prediction','Count'],['unknown',113],['possibly_damaging',912],['probably_damaging',1668],['benign',2260]]));

          google.visualization.events.addListener(polyphen_pie, 'select', function() {
            polyphen_table.setSelection(polyphen_pie.getSelection());
          });
          google.visualization.events.addListener(polyphen_table, 'select', function() {
            polyphen_pie.setSelection(polyphen_table.getSelection());
          });
        var chr_bar = drawBar('chr_bar', 'Variants by chromosome', google.visualization.arrayToDataTable([['Chromosome','Count'],['1',48653],['2',52378],['3',41763],['4',36159],['5',37575],['6',42021],['7',33469],['8',32228],['9',28769],['10',31700],['11',31182],['12',30230],['13',23359],['14',20569],['15',20400],['16',22425],['17',21033],['18',18556],['19',16149],['20',17590],['21',9947],['22',10663],['MT',219],['X',23602],['Y',1775]]), {legend: {position: "none"}});
var chr_table = drawTable('chr_table', 'Variants by chromosome', google.visualization.arrayToDataTable([['Chromosome','Count'],['1',48653],['2',52378],['3',41763],['4',36159],['5',37575],['6',42021],['7',33469],['8',32228],['9',28769],['10',31700],['11',31182],['12',30230],['13',23359],['14',20569],['15',20400],['16',22425],['17',21033],['18',18556],['19',16149],['20',17590],['21',9947],['22',10663],['MT',219],['X',23602],['Y',1775]]));

          google.visualization.events.addListener(chr_bar, 'select', function() {
            chr_table.setSelection(chr_bar.getSelection());
          });
          google.visualization.events.addListener(chr_table, 'select', function() {
            chr_bar.setSelection(chr_table.getSelection());
          });
        var protein_bar = drawBar('protein_bar', 'Position in protein', google.visualization.arrayToDataTable([['Position in protein (percentile)','Count'],['00-10%',1033],['10-20%',989],['20-30%',1013],['30-40%',1045],['40-50%',1042],['50-60%',1027],['60-70%',1009],['70-80%',1029],['80-90%',1049],['90-100%',1124]]), {hAxis: {title: "Position in protein (percentile)", textStyle: {fontSize: 10}}, legend: {position: "none"}});

    }
    
    function drawPie(id, title, data, options) {    
      var pie = new google.visualization.PieChart(document.getElementById(id));
      pie.draw(data, options);
      return pie;
    }
    function drawBar(id, title, data, options) {
      var bar = new google.visualization.ColumnChart(document.getElementById(id));
      bar.draw(data, options);
      return bar;
    }
    function drawTable(id, title, data) {
      var table = new google.visualization.Table(document.getElementById(id));
      table.draw(data, null);
      return table;
    }
    function drawLine(id, title, data, options) {
      var line = new google.visualization.LineChart(document.getElementById(id));
      line.draw(data, options);
      return line;
    }
    function drawArea(id, title, data, options) {
      var area = new google.visualization.AreaChart(document.getElementById(id));
      area.draw(data, options);
      return area;
    }
    google.setOnLoadCallback(init);
  </script>
  
  
  <style type="text/css">
    body {
      font-family: arial, sans-serif;
      margin: 0px;
      padding: 0px;
    }
    
    a {color: #36b;}
    a.visited {color: #006;}
    
    .stats_table {
      margin: 5px;
    }
    
    td {
      padding: 5px;
    }

    th.gradient {
      height: auto;
    }

    .stats_table tr:nth-child(odd) {
      background-color: #f0f0f0;
    }
    
    h3 {
      color: #666;
    }
    
    .masthead {
      background-color: white;
      color: rgb(204, 221, 255);
      height: 80px;
      width: 100%;
      padding: 0px;
    }
    
    .main {
      padding: 10px;
    }
    
    .gradient {
      background: #333366; /* Old browsers */
      background: -moz-linear-gradient(left,  #333366 0%, #ffffff 100%); /* FF3.6+ */
      background: -webkit-gradient(linear, left top, right top, color-stop(0%,#333366), color-stop(100%,#ffffff)); /* Chrome,Safari4+ */
      background: -webkit-linear-gradient(left,  #333366 0%,#ffffff 100%); /* Chrome10+,Safari5.1+ */
      background: -o-linear-gradient(left,  #333366 0%,#ffffff 100%); /* Opera 11.10+ */
      background: -ms-linear-gradient(left,  #333366 0%,#ffffff 100%); /* IE10+ */
      background: linear-gradient(to right,  #333366 0%,#ffffff 100%); /* W3C */
      filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#333366', endColorstr='#ffffff',GradientType=1 ); /* IE6-9 */
      
      padding: 0px;
      height: 80px;
      width: 700px;
    }
    
    .main_content {
      margin-left: 300px;
    }
    
    .sidemenu {
      width: 260px;
      position: fixed;
      border-style: solid;
      border-width: 2px;
      border-color: rgb(51, 51, 102);
    }
    
    .sidemenu_head {
      width: 250px;
      background-color: rgb(51, 51, 102);
      color: rgb(204, 221, 255);
      padding: 5px;
    }
    
    .sidemenu_body {
      width: 250px;
      padding: 5px;
    }
  </style>
</head>
<body>
<div id="masthead" class="masthead">
  <div style="float: left; display: inline; padding: 10px; height: 80px;">
    <a href="http://www.ensembl.org/"><img src="http://static.ensembl.org/i/e-ensembl.png" width="185" height="45" alt="Ensembl"></a>
  </div>
  
  <div style="float: right; display: inline; height: 80px; background: white; padding: 10px;">
    <a href="http://www.ensembl.org/vep"><img src="http://www.ensembl.org/img/vep_logo.png" width="111" height="50" alt="VEP"></a>
  </div>
  <div class="gradient">
  </div>
</div>
<div class="main">
<div class="sidemenu"><div class="sidemenu_head">Links</div><div class="sidemenu_body"><ul><li><a href="#masthead">Top of page</a></li><li><a href="#run_stats">VEP run statistics</a></li><li><a href="#gen_stats">General statistics</a></li><li><a href="#var_class">Variant classes</a></li><li><a href="#var_cons">Consequences (most severe)</a></li><li><a href="#consequences">Consequences (all)</a></li><li><a href="#coding">Coding consequences</a></li><li><a href="#sift">SIFT summary</a></li><li><a href="#polyphen">PolyPhen summary</a></li><li><a href="#chr">Variants by chromosome</a></li><li><a href="#protein">Position in protein</a></li></ul></div></div><div class='main_content'><h3 id="run_stats">VEP run statistics</h3><table class="stats_table"><tr><td>VEP version (API)</td><td> 93 (93)</td></tr><tr><td>Annotation sources</td><td>Cache: /projects/pynnotator/pynnotator/data/vep_data/homo_sapiens/93_GRCh37</td></tr><tr><td>Species</td><td>homo_sapiens</td></tr><tr><td>Command line options</td><td><pre>--assembly GRCh37 --biotype --cache --coding_only --dir /projects/pynnotator/pynnotator/data/vep_data --force_overwrite --fork 4 --format vcf --input_file sanity_check/sorted.vcf --no_intergenic --no_progress --numbers --offline --output_file vep/vep.output.vcf --pick --plugin dbNSFP,/projects/pynnotator/pynnotator/data/dbnsfp/dbNSFP3.5a.txt.gz,genename,cds_strand,refcodon,codonpos,codon_degeneracy,Ancestral_allele,AltaiNeandertal,Denisova,Ensembl_geneid,Ensembl_transcriptid,Ensembl_proteinid,aapos,SIFT_score,SIFT_converted_rankscore,SIFT_pred,Uniprot_acc_Polyphen2,Uniprot_id_Polyphen2,Uniprot_aapos_Polyphen2,Polyphen2_HDIV_score,Polyphen2_HDIV_rankscore,Polyphen2_HDIV_pred,Polyphen2_HVAR_score,Polyphen2_HVAR_rankscore,Polyphen2_HVAR_pred,LRT_score,LRT_converted_rankscore,LRT_pred,LRT_Omega,MutationTaster_score,MutationTaster_converted_rankscore,MutationTaster_pred,MutationTaster_model,MutationTaster_AAE,MutationAssessor_UniprotID,MutationAssessor_variant,MutationAssessor_score,MutationAssessor_score_rankscore,MutationAssessor_pred,FATHMM_score,FATHMM_converted_rankscore,FATHMM_pred,PROVEAN_score,PROVEAN_converted_rankscore,PROVEAN_pred,Transcript_id_VEST3,Transcript_var_VEST3,VEST3_score,VEST3_rankscore,MetaSVM_score,MetaSVM_rankscore,MetaSVM_pred,MetaLR_score,MetaLR_rankscore,MetaLR_pred,Reliability_index,M-CAP_score,M-CAP_rankscore,M-CAP_pred,REVEL_score,REVEL_rankscore,MutPred_score,MutPred_rankscore,MutPred_protID,MutPred_AAchange,MutPred_Top5features,CADD_raw,CADD_raw_rankscore,CADD_phred,DANN_score,DANN_rankscore,fathmm-MKL_coding_score,fathmm-MKL_coding_rankscore,fathmm-MKL_coding_pred,fathmm-MKL_coding_group,Eigen_coding_or_noncoding,Eigen-raw,Eigen-phred,Eigen-PC-raw,Eigen-PC-phred,Eigen-PC-raw_rankscore,GenoCanyon_score,GenoCanyon_score_rankscore,integrated_fitCons_score,integrated_fitCons_score_rankscore,integrated_confidence_value,GM12878_fitCons_score,GM12878_fitCons_score_rankscore,GM12878_confidence_value,H1-hESC_fitCons_score,H1-hESC_fitCons_score_rankscore,H1-hESC_confidence_value,HUVEC_fitCons_score,HUVEC_fitCons_score_rankscore,HUVEC_confidence_value,GERP++_NR,GERP++_RS,GERP++_RS_rankscore,phyloP100way_vertebrate,phyloP100way_vertebrate_rankscore,phyloP20way_mammalian,phyloP20way_mammalian_rankscore,phastCons100way_vertebrate,phastCons100way_vertebrate_rankscore,phastCons20way_mammalian,phastCons20way_mammalian_rankscore,SiPhy_29way_pi,SiPhy_29way_logOdds,SiPhy_29way_logOdds_rankscore,1000Gp3_AC,1000Gp3_AF,1000Gp3_AFR_AC,1000Gp3_AFR_AF,1000Gp3_EUR_AC,1000Gp3_EUR_AF,1000Gp3_AMR_AC,1000Gp3_AMR_AF,1000Gp3_EAS_AC,1000Gp3_EAS_AF,1000Gp3_SAS_AC,1000Gp3_SAS_AF,TWINSUK_AC,TWINSUK_AF,ALSPAC_AC,ALSPAC_AF,ESP6500_AA_AC,ESP6500_AA_AF,ESP6500_EA_AC,ESP6500_EA_AF,ExAC_AC,ExAC_AF,ExAC_Adj_AC,ExAC_Adj_AF,ExAC_AFR_AC,ExAC_AFR_AF,ExAC_AMR_AC,ExAC_AMR_AF,ExAC_EAS_AC,ExAC_EAS_AF,ExAC_FIN_AC,ExAC_FIN_AF,ExAC_NFE_AC,ExAC_NFE_AF,ExAC_SAS_AC,ExAC_SAS_AF,ExAC_nonTCGA_AC,ExAC_nonTCGA_AF,ExAC_nonTCGA_Adj_AC,ExAC_nonTCGA_Adj_AF,ExAC_nonTCGA_AFR_AC,ExAC_nonTCGA_AFR_AF,ExAC_nonTCGA_AMR_AC,ExAC_nonTCGA_AMR_AF,ExAC_nonTCGA_EAS_AC,ExAC_nonTCGA_EAS_AF,ExAC_nonTCGA_FIN_AC,ExAC_nonTCGA_FIN_AF,ExAC_nonTCGA_NFE_AC,ExAC_nonTCGA_NFE_AF,ExAC_nonTCGA_SAS_AC,ExAC_nonTCGA_SAS_AF,ExAC_nonpsych_AC,ExAC_nonpsych_AF,ExAC_nonpsych_Adj_AC,ExAC_nonpsych_Adj_AF,ExAC_nonpsych_AFR_AC,ExAC_nonpsych_AFR_AF,ExAC_nonpsych_AMR_AC,ExAC_nonpsych_AMR_AF,ExAC_nonpsych_EAS_AC,ExAC_nonpsych_EAS_AF,ExAC_nonpsych_FIN_AC,ExAC_nonpsych_FIN_AF,ExAC_nonpsych_NFE_AC,ExAC_nonpsych_NFE_AF,ExAC_nonpsych_SAS_AC,ExAC_nonpsych_SAS_AF,gnomAD_exomes_AC,gnomAD_exomes_AN,gnomAD_exomes_AF,gnomAD_exomes_AFR_AC,gnomAD_exomes_AFR_AN,gnomAD_exomes_AFR_AF,gnomAD_exomes_AMR_AC,gnomAD_exomes_AMR_AN,gnomAD_exomes_AMR_AF,gnomAD_exomes_ASJ_AC,gnomAD_exomes_ASJ_AN,gnomAD_exomes_ASJ_AF,gnomAD_exomes_EAS_AC,gnomAD_exomes_EAS_AN,gnomAD_exomes_EAS_AF,gnomAD_exomes_FIN_AC,gnomAD_exomes_FIN_AN,gnomAD_exomes_FIN_AF,gnomAD_exomes_NFE_AC,gnomAD_exomes_NFE_AN,gnomAD_exomes_NFE_AF,gnomAD_exomes_SAS_AC,gnomAD_exomes_SAS_AN,gnomAD_exomes_SAS_AF,gnomAD_exomes_OTH_AC,gnomAD_exomes_OTH_AN,gnomAD_exomes_OTH_AF,gnomAD_genomes_AC,gnomAD_genomes_AN,gnomAD_genomes_AF,gnomAD_genomes_AFR_AC,gnomAD_genomes_AFR_AN,gnomAD_genomes_AFR_AF,gnomAD_genomes_AMR_AC,gnomAD_genomes_AMR_AN,gnomAD_genomes_AMR_AF,gnomAD_genomes_ASJ_AC,gnomAD_genomes_ASJ_AN,gnomAD_genomes_ASJ_AF,gnomAD_genomes_EAS_AC,gnomAD_genomes_EAS_AN,gnomAD_genomes_EAS_AF,gnomAD_genomes_FIN_AC,gnomAD_genomes_FIN_AN,gnomAD_genomes_FIN_AF,gnomAD_genomes_NFE_AC,gnomAD_genomes_NFE_AN,gnomAD_genomes_NFE_AF,gnomAD_genomes_OTH_AC,gnomAD_genomes_OTH_AN,gnomAD_genomes_OTH_AF,clinvar_rs,clinvar_clnsig,clinvar_trait,clinvar_golden_stars,Interpro_domain,GTEx_V6p_gene,GTEx_V6p_tissue --polyphen b --sift b --symbol --total_length --vcf</pre></td></tr><tr><td>Start time</td><td>2018-09-05 00:40:14</td></tr><tr><td>End time</td><td>2018-09-05 00:45:18</td></tr><tr><td>Run time</td><td>304 seconds</td></tr><tr><td>Input file</td><td>sanity_check/sorted.vcf</td></tr><tr><td>Output file</td><td>vep/vep.output.vcf</td></tr></table><h3 id="gen_stats">General statistics</h3><table class="stats_table"><tr><td>Lines of input read</td><td>652414</td></tr><tr><td>Variants processed</td><td>652414</td></tr><tr><td>Variants filtered out</td><td>0</td></tr><tr><td>Novel / existing variants</td><td>-</td></tr><tr><td>Overlapped genes</td><td>5680</td></tr><tr><td>Overlapped transcripts</td><td>5879</td></tr><tr><td>Overlapped regulatory features</td><td>-</td></tr></table><hr/><h3 id="var_class">Variant classes</h3><div id="var_class_pie" style="width: 800px; height: 200px">&nbsp;</div><div id="var_class_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="var_cons">Consequences (most severe)</h3><div id="var_cons_pie" style="width: 800px; height: 400px">&nbsp;</div><div id="var_cons_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="consequences">Consequences (all)</h3><div id="consequences_pie" style="width: 800px; height: 400px">&nbsp;</div><div id="consequences_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="coding">Coding consequences</h3><div id="coding_pie" style="width: 800px; height: 400px">&nbsp;</div><div id="coding_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="sift">SIFT summary</h3><div id="sift_pie" style="width: 800px; height: 200px">&nbsp;</div><div id="sift_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="polyphen">PolyPhen summary</h3><div id="polyphen_pie" style="width: 800px; height: 200px">&nbsp;</div><div id="polyphen_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr">Variants by chromosome</h3><div id="chr_bar" style="width: 800px; height: 200px">&nbsp;</div><div id="chr_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="protein">Position in protein</h3><div id="protein_bar" style="width: 800px; height: 200px">&nbsp;</div></div>
</div></body>
</html>
